EGO Forum Enhancement.user.js

```diff
@@ -318,5 +424,8 @@ function handleLeadership() {
     if (url.match(/^https:\/\/www\.edgegamers\.com\/forums\/?$/))  // Forums List
         handleForumsList();
 
+    if (url.match(/^https:\/\/www\.edgegamers\.com\/application\/\d+\/?$/))  // Application Page
+        handleApplicationPage();
+
```

Comment:
All of the ifs from line 421-427 (one more block above than I've highlighted here) should be swapped for else ifs, those regexes are mutually exclusive.